__tests__/test-data.test.js

```diff
@@ -77,5 +77,41 @@ describe("GET /api", () => {
         }
       });
   });
+});
 
+describe("GET /api/articles/:article_id", () => {
+  test("200: responds with status code 200 with valid request", () => {
+    return request(app).get("/api/articles/1").expect(200);
+  });
+  test("returns an object of a specific article to the client", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .then((response) => {
+        const article = response.body.article;
+        expect(typeof article.author).toBe("string");
+        expect(typeof article.title).toBe("string");
+        expect(typeof article.article_id).toBe("number");
```

Comment:
You've requested article 1, so need to check you've got that one. At the moment, this will match any object and you can't be sure you've got the correct one.